packages/data/addon/services/navi-dimension.ts

```diff
@@ -47,7 +44,7 @@ class DimensionCache extends Cache<NaviDimensionResponse> {
       });
       return cannonicalName;
     }
-    super(getDimensionCacheId, MAX_CACHE_SIZE, CACHE_TIMEOUT);
```

Comment:
We should add these to the config
https://github.com/yahoo/yavin/blob/018103098cbb47f7b932a38edda5630271e89714/packages/data/config/environment.js#L7
and declare them here
https://github.com/yahoo/yavin/blob/04e324c3a0d93737c7fb5f533209aac1bb65cbc9/packages/data/config/navi-config.d.ts#L42

```
dimensionCache: {
	size: 10,
	timeoutMs: 60 * 60 * 1000
}
```